tests/module_1/test_meteo_api.py

```diff
@@ -1,7 +1,32 @@
 """ This is a dummy example to show how to import code from src/ for testing"""
 
-from src.module_1.module_1_meteo_api import main
+
+import unittest
+import pandas as pd
+
+from src.module_1.module_1_meteo_api import get_yearly_mean_std, VARIABLES
+
+# flake8: noqa E501
 
 
 def test_main():
-    raise NotImplementedError
\ No newline at end of file
+    variable = VARIABLES.split(",")[-1]
+    data_dict = {
+        "time": ["1950-01-01", "1950-02-01", "1975-02-01", "1975-02-03"],
+        f"{variable}_CMCC_CM2_VHR4": [10, 12, 10, 10],
+    }
+
+    variable_dict = {variable: "°C"}
+
+    result = get_yearly_mean_std(data_dict, variable_dict)
+
+    expected_result = {
```

Comment:
You see with the test is easier to understand what the function does. Here my challenge is that you are not doing the mean + std deviation across climate models but within a model across the year.

 I understand that if there are multiple models we would be calculating the yearly mean + std after stacking all models together, right? Than the test with two models should be more convenient.